docs/apis/jsonrpc-apis.md

```diff
@@ -39,10 +42,11 @@ You'll need to use your own programming environments' capabilities (libraries, t
 connect via HTTP, WS or IPC to a `geth` node configured with the above flags and you'll
 need to speak [JSON-RPC](https://www.jsonrpc.org/specification) on all transports. You
 can reuse the same connection for multiple requests!
-
-**Note: Please understand the security implications of opening up an HTTP/WS based
-transport before doing so! Hackers on the internet are actively trying to subvert
-Ethereum nodes with exposed APIs! Further, all browser tabs can access locally
-running web servers, so malicious web pages could try to subvert locally available
-APIs!**
-
+Here you can check the available [JSON-RPC calls](https://playground.open-rpc.org/?schemaUrl={{ openrpc_latest_schema_endpoint }}){: target=_blank }.
+
+!!! attention
```

Comment:
Is `!!! ` something special?

Reply:
Yes, it creates tooltips like in the below image. Here is the [docs](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) for it.

![image](https://user-images.githubusercontent.com/802700/104583975-80a8cb80-566a-11eb-8381-aeb2da33ae9d.png)

Reply:
That looks really nice, but my concern is that it's not generic (or Github-flavored) markdown, which I'd generally like to stick with if possible for portability's sake. 

Can we instead approximate this rendering with something like

> __:warning: Attention__
> Please understand the security...

Reply:
They do look really nice though... maybe I could be convinced...

Reply:
I really like them and I can't decide.
If we suggest everyone to read the docs from within the site, and based on that `!!! <attention>` for devs reading the docs directly the word next to `!!!` describes the level/intention of the message below, we might want to consider using it.

On the other hand, what you suggest above is a good alternative as well

Reply:
Your alternative serves our scope as well, but the material one is very nice 👍 

![image](https://user-images.githubusercontent.com/802700/104604599-32a1c100-5686-11eb-861f-0a814ba153dc.png)

Reply:
We decided to continue with `!!! <attention>` way. If needed to be replaced later we can easily find all occurrences and replace them.